build.gradle

```diff
@@ -10,7 +9,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.1'
+        classpath 'com.android.tools.build:gradle:3.6.0'
```

Comment:
3.6.0 is minimum with this type of upload, otherwise withXml needs to be used. I plan to upgrade further more in the next PRs